src/gflags.cc

```diff
@@ -1163,8 +1189,8 @@ string CommandLineFlagParser::ProcessFromenvLocked(const string& flagval,
     }
 
     const string envname = string("FLAGS_") + string(flagname);
-	string envval;
-	if (!SafeGetEnv(envname.c_str(), envval)) {
+  string envval;
```

Comment:
When correcting the indentation, I looks like it should be 4 spaces instead than just 2?